angular-hometask/src/app/modules/courses-page/entities/validators/date-validator.directive.ts

```diff
@@ -0,0 +1,8 @@
+import { AbstractControl } from '@angular/forms';
+
+const dateReg = new RegExp('^([0]?[1-9]|[1|2][0-9]|[3][0|1])[/]([0]?[1-9]|[1][0-2])[/]([0-9]{4})$');
+
+export function dateValidator(control: AbstractControl): { [key: string]: any } | null {
```

Comment:
Don't need to write | null, since this type included in everything

Reply:
516f5acd33596cb71d74107c1c926b967a830522